components/media-toolbar/index.tsx

```diff
@@ -40,6 +49,11 @@ export const MediaToolbar: React.FC<MediaToolbarProps> = ({
 	onRemove,
 	isOptional = false,
 	id,
+	labels = {
+		add: __('Add Image', '10up-block-components'),
+		remove: __('Remove Image', '10up-block-components'),
+		replace: __('Replace Image', '10up-block-components'),
+	},
```

Comment:
@Firestorm980 The downside to this way of handling the default values is that if you just want to customize the label for one of them you loose the default values of the other two. 

Also if we want to add a new label in te future we cause breakages for anyone using the custom labels because the new option would be undefined. 

Because of that I think the following would be more safe:


```js
const DEFAULT_LABELS = {
    add: __('Add Image', '10up-block-components'),
    remove: __('Remove Image', '10up-block-components'),
    replace: __('Replace Image', '10up-block-components'),
}

function ( { labels = {} } ) {

    const mergedLabels = {
        ...DEFAULT_LABELS,
        ...labels,
    };
}
```